pages/common/git-grep.md

```diff
@@ -0,0 +1,19 @@
+# git-grep
+
+> Find strings inside files as they existed anywhere in repo history. Accepts a lot of the same flags as regular grep. Learn more here: https://git-scm.com/docs/git-grep.
+
+- Search for a string in working directory:
+
+`git grep {{search_string}}`
+
+- Search for string in files matching a pattern in working directory:
+
+`git grep {{search_string}} -- {{file_glob_pattern}}`
+
+- Search for a string in working directory, including submodules:
```

Comment:
Perhaps this will read better?

```suggestion
- Search for a string in the current working directory, including submodules:
```

Reply:
One more. "Search for a string in non-ignored files in the current directory and all submodules:"